connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -3183,9 +3184,9 @@ class SparkConnectPlanner(
       case StreamingQueryManagerCommand.CommandCase.ACTIVE =>
         val active_queries = session.streams.active
         respBuilder.getActiveBuilder.addAllActiveQueries(
-          active_queries
-            .map(query => buildStreamingQueryInstance(query))
-            .toIterable
+          immutable.ArraySeq
+            .unsafeWrapArray(active_queries
+              .map(query => buildStreamingQueryInstance(query)))
```

Comment:
nit. In this case, one liner could be better and consistent with other parts.
```
.unsafeWrapArray(active_queries.map(query => buildStreamingQueryInstance(query)))
```

Reply:
The code in the connect module must use scalafmt... the result of this scalafmt is mandatory, otherwise, the lint will fail the check :(

Reply:
Oh, got it. Thank you, @LuciferYang .